cellxgene_schema_cli/cellxgene_schema/remove_labels.py

```diff
@@ -30,23 +30,33 @@ def remove_labels(self):
             if component is None:
                 continue
 
+            component_def = self.schema_def["components"][component_name]
```

Comment:
minor refactor here to just use a variable name instead of referencing `self.schema_def["components"][component_name]` seven times